index.html

```diff
@@ -0,0 +1,918 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
- Replace the default viewport meta tag with the responsive one below :arrow_double_down: 
`<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`

Reply:
Thanks! Added the responsive meta